src/locale/cs.ts

```diff
@@ -0,0 +1,94 @@
+	"validationName": "Popis",
+	"validationSeverity": "Vážnost",
+	"errorConnectionLost": "Chyba: připojení k pracovnímu prostoru bylo přerušeno.",
+	"diagramTab": "Diagramy",
+	"modalNewElemExistsError": "Koncept s tímto jménem již ve vybraném slovníku existuje.",
+	"addAltLabelPlaceholder": "Nový synonym...",
+	"setAsDisplayName": "Nastavit jako zobrazované jméno",
+	"conforms": "Pracovní prostor je podle serveru validní",
+	"validationError": "Zpráva",
+	"validationReload": "Obnovit",
+	"validation": "Validace",
+	"setStereotypeUML": "Nastavit typový stereotyp",
+	"setStereotypeData": "Nastavit OntoUML stereotyp",
+	"noStereotypeUML": "Žádný typový stereotyp",
+	"noStereotypeData": "Žádný OntoUML stereotyp",
+	"sourceCardinality": "Kardinalita u zdroje",
+	"targetCardinality": "Kardinalita u cíle",
+	"loading": "Načítání...",
+	"validationLoadingError": "Připojení k validačnímu serveru se nezdařilo. Prosím, zkuste to znovu.",
+	"pleaseReload": "Chyba: pracovní prostor nebyl nalezen. Prosím, zkontrolujte IRI pracovního prostoru.",
+	"spreadConnections": "Zobrazit všechny vztahy na plátně",
+	"connections": "Vztahy",
+	"linkType": "Typ",
+	"searchStereotypes": "Hledat...",
+	"untitled": "Nový diagram",
+	"none": "",
+	"cancel": "Zrušit",
+	"menuPanelHelp": "Nápověda",
+	"zoomIn": "Přiblížit",
+	"zoomOut": "Oddálit",
+	"menuPanelView": "Vzhled",
+	"menuPanelCenterView": "Vycentrovat pohled",
+	"menuPanelZoom": "Obnovit přiblížení",
+	"menuPanelFitContent": "Zobrazit celý diagram najednou",
+	"close": "Zavřít",
+	"confirm": "Potvrdit",
+	"savedChanges": "✔ Úspěšně uloženo.",
+	"stableConnection": "Jste připojen/a k databázi.",
+	"brokenConnection": "Vaše připojení k databázi bylo přerušeno.",
+	"description": "Popis",
+	"vocabularyNotFound": "URL definice slovníků chybí nebo není správné",
+	"validate": "Validovat",
+	"modalRemovePackageItemTitle": "Smazat koncept?",
+	"modalRemoveDiagramTitle": "Smazat diagram?",
+	"modalRemoveDiagramDescription": "Opravdu chcete smazat diagram?",
+	"connectionVia": "přes vztah",
+	"connectionTo": "ke",
+	"connectionFrom": "od",
+	"diagram": "v diagramu",
+	"modalRemovePackageItemDescription": "Opravdu chcete smazat concept?",
+	"definitions": "Popisy",
+	"representationCompact": "Přepnout na Compact pohled",
+	"representationFull": "Přepnout na Full pohled",
+	"showStereotypes": "Ukázat stereotypy",
+	"showIncompatibleLinks": "Ukázat nekompatibilní vztahy",
+	"hideStereotypes": "Skrýt stereotypy",
+	"modalNewElemDescription": "Jméno pro nový koncept (nelze později změnit!):",
+	"modalNewElemTitle": "Vytvořit nový koncept",
+	"modalNewLinkTitle": "Vytvořit nový vztah",
+	"changelog": "Changelog",
+	"modalNewLinkDescription": "Vyberte vztah z následujících možností:",
+	"labels": "Jméno",
+	"delete": "Smazat",
+	"fileProjectSettingsDescriptions": "Popis",
+	"help": "Nápověda",
+	"modalNewElemError": "Jméno konceptu je povinné.",
+	"selectPackage": "Koncept se vytvoří v:",
```

Comment:
TermIt uses the word "Pojem" in Czech. Although both are defendable, "Pojem" looks more "friendly" than "koncept". I propose to change the wording of all occurrences of "Koncept" to "Pojem"